docker/run-apache2.sh

```diff
@@ -3,6 +3,8 @@
 _DECK_HOST=$DECK_HOST
 _DECK_PORT=$DECK_PORT
 _API_HOST=$API_HOST
+_DECK_CERT=$DECK_CERT
```

Comment:
These are paths, right? It might be best to name these `DECK_CERT_PATH` & `DECK_KEY_PATH`, in case we want to enable putting the keys directly into the env var in the future.